plugins/katello/3.14/installation/index.md

```diff
@@ -170,6 +170,65 @@ arguments. Multiple entries can be entered via
         reverse: true
         contact: hostmaster.example.com.
 
+## Tuning options
+The Foreman installer supports automatic tuning of your environment using predefined tuning profiles.  These tuning profiles are the result of a culmination of extensive learning from Foreman environments deployed at scale in large user environments.
+
+When the foreman-installer is run, it is deployed with a predefined tuning profile named `default`. The definition of the `default` profile can be found in `/usr/share/foreman-installer/config/foreman.hiera/tuning/common.yaml`.
+
+Other than the default tuned profile, foreman-installer supports 4 different tuning profiles:
+
+  * medium
+  * large
+  * extra-large
+  * extra-extra-large
+
+Definitions of various tuning profiles can be found in this directory `/usr/share/foreman-installer/config/foreman.hiera/tuning/sizes/`.  Based on your environment needs, use one of the tuning profiles (`medium`, `large`, `extra-large`, `extra-extra-large`) in the installer.  For example, `medium` profile can be applied like:
+
+```bash
+foreman-installer --tuning medium
+```
+
+To reset to the default profile:
+
+```bash
+foreman-installer --tuning default
+```
+
+**Note**
```

Comment:
I'd move all these notes to the bottom of the section, below the table. My reasoning is that docs should go from most relevant to least relevant. Using `custom-hiera.yaml` is unsupported and you are expected to be an advanced user already. That means it should be at the bottom.

Reply:
Good point about not emphasizing too much on `custom-hiera.yaml` for entry-level. Now I created a new sub-section.  Moved the note on custom-hiera.yaml to the end of the first section (not to the very end as per your comment) because the second sub-section is mostly for choosing the right tuned profile.